aws/src/main/java/org/apache/iceberg/aws/s3/S3InputFile.java

```diff
@@ -49,7 +47,6 @@ public long getLength() {
 
   @Override
   public SeekableInputStream newStream() {
-    return new S3InputStream(client(), uri(), awsProperties());
+    return new S3InputStream(client(), uri(), awsProperties(), metrics());
   }
-
```

Comment:
Nit: unnecessary whitespace change.

Reply:
This is just actually fixing an existing whitespace issue.